build.js

```diff
@@ -23,7 +23,7 @@ var noStrict = 'no-strict' in buildArgsAsObject;
 var noSVGExport = 'no-svg-export' in buildArgsAsObject;
 var requirejs = 'requirejs' in buildArgsAsObject ? 'requirejs' : false;
 var sourceMap = 'sourcemap' in buildArgsAsObject;
-
+var buildFast = 'fast' in buildArgsAsObject;
```

Comment:
I have no issue with a build fast step, but want to note that I don't think the regular build is slow.
Because we can have a 'build' stage, we only need to run once anyway.

Reply:
so the `fast`build is just to skip the uglify and multiple file creation and uglify-js installation too.
Also i will use it in build:watch so that when i updated files when developing, the test website is update a couple of seconds faster.

all the tests runs on the non minified build, so there is no need to minify it.
Is important to test the minification in the lint stage because uglify can fail sometimes with something that lint does not catch.

Reply:
Also each stage start a fresh container, so i have to build each stage.
I do not think caching the dist folder ( that change each PR  and so gets updated by travis ) is a good thing, so for now we are stuck with a build for each stage and job, so i think the faster the better